interfaces/cython/cantera/ck2yaml.py

```diff
@@ -2077,6 +2133,13 @@ def main(argv):
               ' must be provided.\nRun "ck2yaml.py --help" to see usage help.')
         sys.exit(1)
 
+    if '--extra' in options:
+        extra_file = options['--extra']
+        if not extra_file.endswith('.yaml') and not extra_file.endswith('.yml'):
+            extra_file += '.yaml'
```

Comment:
I'm not sure this is necessary. Why did you decide to include this? We don't do this for other files here (e.g., adding `.dat`). If this can be removed, these several lines can be just `extra_file = options.get('--extra')`, I think